From 0a05e169365b451af9668c2033c355bc8ceb4d78 Mon Sep 17 00:00:00 2001
From: Alex Marginean <alexandru.marginean@nxp.com>
Date: Mon, 10 Dec 2018 13:49:44 +0200
Subject: [PATCH 399/706] arm64: dts: fsl: ls1028a: disble EEE on SGMII PHY on
 RDB

This causes traffic loss under light load on back 2 back links.

Signed-off-by: Alex Marginean <alexandru.marginean@nxp.com>
(cherry picked from commit 8a7673b9131aa23310a14cca8f84eb1b232d5205)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
index dda99a19e954..0b9a6da9584d 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a-rdb.dts
@@ -245,6 +245,8 @@
 &mdio0 {
 	sgmii_phy0: ethernet-phy@2 {
 		reg = <0x2>;
+		// Atheros PHY on RDB seems to cause traffic loss @ 1Gbps w EEE
+		eee-broken-1000t;
 	};
 
 	qsgmii_phy1: ethernet-phy@4 {
-- 
2.17.1

